build/build.xml

```diff
@@ -835,6 +779,24 @@
     <exec executable="./linux/work/arduino" spawn="false" failonerror="true"/>
   </target>
 
+  <target name="build-arduino-builder" unless="no_arduino_builder">
+    <delete dir="${staging_folder}/arduino-builder-${platform}" includeemptydirs="true"/>
+    <mkdir dir="${staging_folder}/arduino-builder-${platform}"/>
+    <antcall target="untar">
+      <param name="archive_file" value="./arduino-builder-${platform}-${ARDUINO-BUILDER-VERSION}.tar.bz2" />
+      <param name="archive_url" value="https://downloads.arduino.cc/tools/arduino-builder-${platform}-${ARDUINO-BUILDER-VERSION}.tar.bz2" />
+      <param name="final_folder" value="${staging_folder}/arduino-builder-${platform}/arduino-builder" />
+      <param name="dest_folder" value="${staging_folder}/arduino-builder-${platform}" />
+    </antcall>
+    <copy file="${staging_folder}/arduino-builder-${platform}/arduino-builder" tofile="linux/work/arduino-builder" />
```

Comment:
This is wrong because the file is always copied into `tofile="linux/work/arduino-builder"` while it should be something like `${staging_folder}/work/arduino-builder`. BTW this is still not correct for macosx, probably the best way is to use `${staging_folder}/work/${staging_hardware_folder}/../arduino-builder` for everything.